stream-chat-android-client/src/main/java/io/getstream/chat/android/client/helpers/AppSettingManager.kt

```diff
@@ -29,15 +29,17 @@ internal class AppSettingManager(private val chatApi: ChatApi) {
     /**
      * Application settings configured in the dashboard and fetched from the backend.
      */
-    private var appSettings: AppSettings = createDefaultAppSettings()
+    private var appSettings: AppSettings? = null
 
     /**
      * Initializes [AppSettingManager] with application settings from the backend.
      */
     fun loadAppSettings() {
-        chatApi.appSettings().enqueue {
-            if (it.isSuccess) {
-                this.appSettings = it.data()
+        if (appSettings == null) {
```

Comment:
We should change the `clear()` implementation to be sure we ask for `AppSettings` after logout

Reply:
Is it safe to only check the app settings once per user login? Or in this way? 

What if the settings change? For example - file/mime types which are allowed can be changed at any point in time and it's important to request them on each app start to make sure we have the latest available config.

Reply:
I believe it would be safe. The change would be applied once the app section would be finished. I this is already the intended behavior, if we add the clear change.

Reply:
@filbabic If it is something that could be updated frequently (more than once per "user session"), it should be provided by the backend as a new event or any other alternative. We can't be doing "pulling" here to be up-to-date